lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -739,4 +740,46 @@ sub estimated_gtmp_for_instrument_data  {
     );
 }
 
+
+sub get_supersede_merged_alignment_results {
```

Comment:
Or, maybe, since this is a method on `Genome::InstrumentData::AlignmentResult::Merged`, we could get away with `get_superseding_results`?